rules/design-system/rule.md

```diff
@@ -0,0 +1,78 @@
+---
+type: rule
+tips: ""
+title: Do you have a Design System?
+seoDescription: Tame visual inconsistencies and streamline team collaboration by
+  building a design system that aligns your product, design, and engineering
+  teams.
+uri: design-system
+authors:
+  - title: Betty Bondoc
+    url: https://www.ssw.com.au/people/betty-bondoc/
+  - title: Jayden Alchin
+    url: https://www.ssw.com.au/people/jayden-alchin/
+related:
+  - design-debt
+  - software-for-ux-design
+  - hand-over-mockups-to-developers
+created: 2025-05-14T15:18:00.000Z
+guid: cf1f0a03-8702-43ae-80d5-f44ac491edee
+---
+When every button looks a little different, spacing feels off, and handoffs between design and dev are painful, you know something’s wrong. These issues lead to inconsistent UIs, longer review cycles, and time wasted on debating visual tweaks instead of building real value.  
```

Comment:
```suggestion
When every button looks a little different, spacing feels off, and handoffs between design and dev are painful, you know something’s wrong. These issues lead to inconsistent UI, longer review cycles, and time wasted debating visual tweaks instead of building real value.  
```